File: c++/C++/nilai.cpp
#include <iostream>
using namespace std;

int main() {
    float tugas,uts,uas,nilaiakhir;

    cout << "Masukkan nilai tugas: ";
    cin >> tugas;
    cout << "Masukkan nilai UTS: ";
    cin >> uts;
    cout << "Masukkan nilai UAS: ";
    cin >> uas;

    nilaiakhir = (0.3 * tugas) + (0.3 * uts) + (0.4 * uas);
    cout << "Nilai Akhir: " << nilaiakhir << endl;
    if(nilaiakhir>=60)
    cout << "status: Lulus" << endl;
    else
    cout << "status: Tidak Lulus" << endl;
    return 0;
}

File: PYTHON/nilai.py
tugas=float(input("Masukkan nilai tugas: "))
uts=float(input("Masukkan nilai UTS: "))
uas=float(input("Masukkan nilai UAS: "))

nilaiakhir = (0.3 * tugas) + (0.3 * uts) + (0.4 * uas)
print("Nilai Akhir:", nilaiakhir)
if nilaiakhir >= 60:
    print("status: Lulus")
else:
    print("status: Tidak Lulus")